### ISSUE TITLE
I made new graphics

### ISSUE THREAD

**Issue by KAMiKAZOW** [DavidoTek/ProtonUp-Qt#64]
For my personal use, I've made a bunch of new graphics that look better in the Steam UI than the current icon. I'm not a pro designer or anything like that. They are CC-BY-SA 4.0 licensed. You may use them if you want.

Steam grid, horizontal:
![ProtonUp-Qt Horizontal](https://user-images.githubusercontent.com/3259076/166148890-40c87f1b-94e9-45e8-851f-01a6079d009b.svg)

Steam grid, vertical / Poster:
![ProtonUp-Qt Icon Poster](https://user-images.githubusercontent.com/3259076/166148891-a68b6d5d-5d49-41e6-bae1-6d8a4f913c58.svg)

Icon (small):
![ProtonUp-Qt Icon](https://user-images.githubusercontent.com/3259076/166148892-aa7f6245-a893-42f5-afac-bd51f615c150.svg)

Logo / Icon (big):
![ProtonUp-Qt Logo](https://user-images.githubusercontent.com/3259076/166148893-117f9d21-b169-4c09-acae-89a9b28488b4.svg)

Created in Inkscape on Steam Deck. Graphics have no intend of being scientifically accurate at all.

**Comment by DavidoTek** [comment]
Thanks for the submission!
I was thinking about changing the icon too, this is a good starting point. I quite like the color scheme, but maybe something could be added to make clear what the app does (like a download arrow, not sure).

**Comment by KAMiKAZOW** [comment]
I was experimenting with an up arrow but it looked too crowded.

**Comment by DavidoTek** [comment]
Okay, cannot think of a way to accomplish this now either.

I will think about it and maybe someone in the community has another idea.

**Comment by onlyreportingissues** [comment]
How about 2 sticking plasters arranged like a cross on the top left part of the green area? An up arrow can be added then on it also & not take too much space, since it gonna be small anyway.

**Comment by DavidoTek** [comment]
Like this?
![grafik](https://user-images.githubusercontent.com/54072917/166885488-54ec0943-ff53-4178-93d3-5b6a886d94c4.png)

My original plan was to draw the structure of a Proton and draw an arrow inside the [Up quarks](https://en.wikipedia.org/wiki/Up_quark). I didn't come up with a better idea than the current logo though.

**Comment by onlyreportingissues** [comment]
Yes, looks cool. And you could add maybe like 3 green arrows
![6ezzdf](https://user-images.githubusercontent.com/103491514/166891386-9c2c4003-08d4-42d4-816f-783b0981e978.jpg)
It's a quick editing. Transparent background is better ofc.

**Comment by KAMiKAZOW** [comment]
And that's not too crowded? Is it even legal to use those arrows under CC-BY-SA 4.0?

**Comment by onlyreportingissues** [comment]
Yea, the arrows are maybe too much.

The plasters can imply that it's still Proton, but with tweaks to "pach the abrasions".

**Comment by DavidoTek** [comment]
Yes, I think that's a bit too much. The Icon should be easily identifiable even when it's small. And with the plaster and the arrows that's unnecessary repeated.
In my opinion the arrow suggests some sort of "up"grade while the plaster rather suggest some sort of patch.
